doc/sphinx/azhelpgen/azhelpgen.py

```diff
@@ -27,7 +27,10 @@ def get_help_files(cli_ctx):
     cli_ctx.invocation.commands_loader.load_command_table([])
     cmd_table = cli_ctx.invocation.commands_loader.command_table
     for command in cmd_table:
-        cli_ctx.invocation.commands_loader.load_arguments(command)
+        try:
+            cli_ctx.invocation.commands_loader.load_arguments(command)        
+        except Exception as ex:
+            print("Skipped '{}' due to '{}'".format(command, ex))
```

Comment:
Instead of simply printing out the exception, could the commands and exceptions be collected and then summarized/outputted at the end?

I believe the CI should still fail upon a exception, so if we can keep the extra robustness while letting the CI fail so that it is more obvious there is a problem that would be for the best.

Reply:
I agree with you. But I am afraid if the CI fails and the document gets updated, the problem might still be ignored.

Reply:
That seems like a big problem. There needs to be a mechanism so that people from our teams are notified of the CI failing. 
It can be difficult to notice from the generated docs themselves.